tests/runtime/probe

```diff
@@ -178,16 +178,6 @@ RUN {{BPFTRACE}} --unsafe -e 'kprobe:ksys_* { @ = probe; printf("progs: "); syst
 EXPECT_REGEX progs: [1-9][0-9]+
 REQUIRES bpftool
 
-# Note: this test may fail if you've installed a new kernel but not rebooted
```

Comment:
Do we not want to leave some kind of record/note as to why we don't have  test for this (perhaps with some of the details in your commit message) ?

Reply:
Eh, on the fence. Hard to discover that information. And we have lots of untested stuff already.

Reply:
Yeah kinda feels like we need some kind of special section that details why we explicitly left out some tests. Make a mental note of this one if you choose to make that PR 😉